scripts/synapse_port_db

```diff
@@ -512,7 +480,7 @@ class Porter(object):
 
         hs = MockHomeserver(self.hs_config, engine, conn, db_pool)
 
-        store = Store(conn, hs)
```

Comment:
wonder if we could use a `DataStores` here for consistency with the app code

Reply:
Yeah, though we only want to include the background update stores rather than building out full data stores. (Currently you could use main_store_class to do this, but when we have future data stores that won't necessarily be true)